### ISSUE TITLE
Add website to event

### ISSUE THREAD

**Issue by marcoroth** [adrienpoly/rubyvideo#350]
A lot of older events have still a website up and it would be cool if the "Visit Website" button would take the user to the website of that event edition rather than the newest edition that lives on the domain.

Example: [EuRuKo 2019](https://www.rubyvideo.dev/events/euruko-2019).

![CleanShot 2024-11-02 at 08 31 37](https://github.com/user-attachments/assets/226b4831-2914-4a54-91e8-c27a2d07db24)

The button takes you to https://euruko.org, but ideally it would point to https://2019.euruko.org.

**Comment by codescaptain** [comment]
@marcoroth  Hello! May I work on this please?

**Comment by marcoroth** [comment]
Hey @codescaptain, sure thing! Let me know if you need any guidance 🙌🏼 

Ideally, we can define the website as an extra key in the `playlists.yml` file and then write it to a database file on the `event` itself.

**Comment by codescaptain** [comment]
hello @marcoroth  ✋🏻  I have some questions: 
 
1.  I am adding a website field for events. Should this field be required, or can it be optional?

2. In the view, if a specific event does not have a link, should the button default to the organization’s main website instead?

**Comment by adrienpoly** [comment]
> 2. In the view, if a specific event does not have a link, should the button default to the organization’s main website instead?

maybe in the model you could have something like that 

```ruby
def website
  self[:website].presence || organisation.website
end
```

so that in the view you always call `event.website`

**Comment by codescaptain** [comment]
@adrienpoly  Thank you for the suggestion! I was initially considering handling this in a helper, but I can proceed with this approach in the model instead. 👍🏻

**Comment by codescaptain** [comment]
I've completed the development. Now, I am also adding the website links for past playlists.

**Comment by marcoroth** [comment]
Resolved via #378
